Client/Frontend/Settings/Clearables.swift

```diff
@@ -43,8 +44,15 @@ class HistoryClearable: Clearable {
         Tab.ChangeUserAgent.clear()
 
         return profile.history.clearHistory().bindQueue(.main) { success in
+            // TODO: Remove clear cache for SDWebImage when we are ready to remove library
```

Comment:
Should we create a ticket for this TODO or it will be done in part2

Reply:
it will be done as part 2
https://mozilla-hub.atlassian.net/browse/FXIOS-4321